crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__RUF200_invalid_author.snap

```diff
@@ -1,13 +1,13 @@
 ---
 source: crates/ruff_linter/src/rules/ruff/mod.rs
 ---
-pyproject.toml:6:84: RUF200 Failed to parse pyproject.toml: invalid type: integer `1`, expected a string
+pyproject.toml:5:11: RUF200 Failed to parse pyproject.toml: a table with 'name' and/or 'email' keys
```

Comment:
The error message isn't ideal but the span should sufficiently prompt users to fix their authors table